docs/telemetry.md

```diff
@@ -1,53 +1,386 @@
+Percona collects usage data to improve its software. The telemetry feature helps us identify popular features, detect problems, and plan future improvements. 
+
+Currently, telemetry is added only to the Percona packages and to Docker images. 
 
 ## What information is collected
 
-Currently, telemetry is added only to the Percona packages and Docker images. It collects only information about the installation environment. Future releases may add additional telemetry metrics.
+Telemetry collects the following information:
+
+* The information about the installation environment when you install the software.
+* The information about the operating system such as name, architecture, the list of Percona packages. See more in the [Telemetry Agent section](#telemetry-agent).
+* The metrics from the database instance. See more in the [percona_pg_telemetry section](#percona_pg_telemetry).
+
+## What is NOT collected
+
+Percona protects your privacy and doesn't collect any personal information about you like database names, user names or credentials or any user-entered values. 
+
+All collected data is anonymous, meaning it can't be traced back to any individual user. To learn more about how Percona handles your data, read the [Percona Privacy statement](https://www.percona.com/privacy-policy).
+
+You control whether to share this information. Participation in this program is completely voluntary. If don't want to share anonymous data, you can [disable telemetry](#disable-telemetry).
+
+## Why telemetry matters
+
+Benefits for Percona:
+
+| Advantages                  | Description                                                                                                                                                         |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| See how people use your software | Telemetry collects anonymous data on how users interact with our software. This tells developers which features are popular, which ones are confusing, and if anything is causing crashes. |
+| Identify issues early       | Telemetry can catch bugs or performance problems before they become widespread. |
+
+Benefits for users in the long run:
+
+| Advantages                | Description                                                                                                         |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------|
+| Faster bug fixes          | With telemetry data, developers can pinpoint issues affecting specific users and prioritize fixing them quickly.  |
```

Comment:
specific users suggests that users can be recognized. Can we rephrase this?

Reply:
Rephrased